neuraxle/data_container.py

```diff
@@ -497,6 +547,61 @@ def concat(self, data_container: DataContainer):
         return self
 
 
+def _pad_or_keep_incomplete_batch(
+        data_container,
+        batch_size,
+        default_value_data_inputs,
+        default_value_expected_outputs
+) -> 'DataContainer':
+    keep_incomplete_batch = isinstance(default_value_data_inputs, AbsentValuesNullObject)
```

Comment:
Misleading variable name, as if we wanted to discard the incomplete minibatch or keep it. Rather, we want to know if we pad it or not. 
```suggestion
    should_pad_right = not isinstance(default_value_data_inputs, AbsentValuesNullObject)
```